### PR TITLE
CodeJam-1 "Curriculum Vitae"

### PR DESCRIPTION
**Pull request by Mudreuski**
CodeJam-1 "Curriculum Vitae"
https://mudreuski.github.io/rsschool-codejam1-cv/
1 commit - feat: create header for cv
2 commit - feat: create contacts for cv
3 commit - feat: create skills for cv
4 commit - feat: create experience for cv
5 commit - feat: create education for cv
6 commit - feat: create English level for cv
7 commit - feat: create footer for cv
8 commit - style: change style.css and index.html
9 commit - style: change index.html

### CODE REVIEW

**Comment by Mudreuski** [comment]
CodeJam-1 "Curriculum Vitae"

1 commit - feat: create header for cv
2 commit - feat: create contacts for cv
3 commit - feat: create skills for cv
4 commit - feat: create experience for cv
5 commit - feat: create education for cv
6 commit - feat: create English level for cv
7 commit - feat: create footer for cv
8 commit - style: change style.css and index.html
9 commit - style: change index.html

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+
+header div{
+    text-align: center;
+}
+
+header img {
+    width: 40%;
+    height: 40%;
+    margin-top: 70px;
+    align-content: center;
```
Comment (Andrei-Zhuk) [inline css/style.css]: no need, for that style

File: css/style.css [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+}
+
+h1,
+footer h2 {
+    text-align: center;
+    color: rgb(184, 245, 245);
+}
+
+h1 {
+    padding: 30px 0px 50px;
```
Comment (Andrei-Zhuk) [inline css/style.css]: it's better to use margin for this reason

File: index.html [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+        <header>
+            <div>
+                <img src="image/photo.jpg" alt="My photo">
+            </div>
+            <h1>CV of Henadzi Mudreuski</h1>
+        </header>
+        <section>
+            <article>
+                <h2>My contacts:</h2>
+                <div>
```
Comment (Andrei-Zhuk) [inline index.html]: it's better to use "ul" tag here for contacts

**Comment by Andrei-Zhuk** [comment]
No issues with commit naming convention and pull request.
Quite semantic code, but in styles it's better to be attached to class names, than to tag names.
The score is 97.
Feel free to merge it.

**Pull request merged**